frontend/common/src/helpers/router.tsx

```diff
@@ -1,9 +1,20 @@
 import { createBrowserHistory, Location, Path, State } from "history";
 import UniversalRouter, { Routes } from "universal-router";
-import React, { useState, useEffect, useMemo, ReactElement } from "react";
+import React, {
+  useState,
+  useEffect,
+  useMemo,
+  ReactElement,
+  useRef,
```

Comment:
Looks like we can removed the `useRef` import now that it is not wrapping the `useApiRoutes`.

Reply:
Good catch!